runtime/rastrace/mt.tdf

```diff
@@ -23,7 +23,9 @@
 Executable=mt
 DATFileName=J9TraceFormat.dat
 
-TraceEntry=Trc_MethodEntry   Overhead=1 Level=5 Group=bytecodeMethods                Template="%.*s.%.*s%.*s bytecode method, this = 0x%zx"
+TraceEntry=Trc_MethodEntry   Obsolete Overhead=1 Level=5 Group=bytecodeMethods       Template="%.*s.%.*s%.*s bytecode method, this = 0x%zx,"
+
+TraceEntry=Trc_MethodEntry   Overhead=1 Level=5 Group=bytecodeMethods                Template="%.*s.%.*s%.*s bytecode method, this = 0x%zx %.*s"
```

Comment:
New tracepoints must be added at the end of files like this (and there's no need to change the template of one that is being obsoleted). See https://github.com/eclipse-openj9/openj9/blob/master/doc/diagnostics/AddingTracepoints.md.